editor/components/post-excerpt/index.js

```diff
@@ -18,15 +18,17 @@ import { editPost } from '../../actions';
 
 function PostExcerpt( { excerpt, onUpdateExcerpt } ) {
 	const onChange = ( event ) => onUpdateExcerpt( event.target.value );
+	const excerptTextareaId = 'editor-post-excerpt';
```

Comment:
IDs must be unique, which conflicts with React componentization where we don't want to assume how many instances of the component are present on the page. For these cases we should use the `withInstanceId` higher-order component to ensure a unique identifier is available:

https://github.com/WordPress/gutenberg/tree/master/components/higher-order/with-instance-id